docs/conf.py

```diff
@@ -208,7 +208,6 @@
     "https://docs.delta.io/latest/delta-update.html#language-python",
     "https://github.com/kedro-org/kedro/blob/main/kedro_technical_charter.pdf",
 
-    # Number section removal, temporarily ignore until 0.18.0 release
```

Comment:
I think actually this comment was referring to all the links below, so we should be able to now remove those from the ignored links.